app/controllers/DeleteAdditionalMetadataController.scala

```diff
@@ -46,7 +51,9 @@ class DeleteAdditionalMetadataController @Inject() (
         Future.failed(new IllegalArgumentException("fileIds are empty"))
       } else {
         for {
-          _ <- customMetadataService.deleteMetadata(fileIds, request.token.bearerAccessToken)
+          displayProperties <- displayPropertiesService.getDisplayProperties(consignmentId, request.token.bearerAccessToken, metadataType)
+          propertiesToDelete: Set[String] = if (metadataType == "closure") Set("ClosureType") else displayProperties.map(_.propertyName).toSet
+          _ <- customMetadataService.deleteMetadata(fileIds, request.token.bearerAccessToken, propertiesToDelete)
```

Comment:
Can we use the same `displayProperties` to delete the closure metadata as well?

Reply:
@vimleshtna does the `displayProperties`table have all the correct closure metadata properties?

Reply:
Yeah, I think so.

```
consignmentapi=# select * from "DisplayProperties" where "Value" = 'Closure';
     PropertyName     |  Attribute   |  Value  | AttributeType 
----------------------+--------------+---------+---------------
 ClosureType          | PropertyType | Closure | text
 DescriptionAlternate | PropertyType | Closure | text
 ClosurePeriod        | PropertyType | Closure | text
 DescriptionClosed    | PropertyType | Closure | text
 TitleAlternate       | PropertyType | Closure | text
 TitleClosed          | PropertyType | Closure | text
 FoiExemptionCode     | PropertyType | Closure | text
 FoiExemptionAsserted | PropertyType | Closure | text
 ClosureStartDate     | PropertyType | Closure | text

```